pandas/io/gbq.py

```diff
@@ -148,15 +168,63 @@ def get_credentials(self):
 
         return credentials
 
+    def get_service_account_credentials(self):
+        from oauth2client.client import SignedJwtAssertionCredentials
+        from os.path import isfile
+
+        try:
+            if isfile(self.private_key):
+                with open(self.private_key) as f:
+                    json_key = json.loads(f.read())
+            else:
+                # ugly hack: 'private_key' field has new lines inside, they break json parser, but we need to preserve them
+                json_key = json.loads(self.private_key.replace('\n', '   '))
```

Comment:
Can we add a check to see if the private key contains new lines before replacing the new line with spaces? My private key does not contain `'\n'`.

```
======================================================================
ERROR: test_should_be_able_to_make_a_connector (pandas.io.tests.test_gbq.TestGBQConnectorServiceAccountKeyPathIntegration)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/tony/pandas-gbq_service_account/pandas/io/tests/test_gbq.py", line 168, in setUp
    self.sut = gbq.GbqConnector(PROJECT_ID, private_key=PRIVATE_KEY_JSON_PATH)
  File "/home/tony/pandas-gbq_service_account/pandas/io/gbq.py", line 143, in __init__
    self.credentials = self.get_credentials()
  File "/home/tony/pandas-gbq_service_account/pandas/io/gbq.py", line 149, in get_credentials
    return self.get_service_account_credentials()
  File "/home/tony/pandas-gbq_service_account/pandas/io/gbq.py", line 181, in get_service_account_credentials
    json_key = json.loads(self.private_key.replace('\n', '   '))
ValueError: Expected object or value
```